watertap/ui/fsapi.py

```diff
@@ -301,7 +348,9 @@ def action_wrapper(**kwargs):
                         "constructor or call `add_action(Actions.export, <function>)` "
                         "on FlowsheetInterface instance."
                     )
-                # run_action will refuse to call the export action directly
+                # clear model_objects dict, since duplicates not allowed
+                self.fs_exp.model_objects = {}
```

Comment:
Incredibly minor suggestion:

```suggestion
                self.fs_exp.model_objects.clear()
```

Reply:
Nice, didn't know that function existed.